### PR TITLE
chore: run format on commit

### PR DESCRIPTION
**Pull request by fisker**
Setup `lint-staged` to format codes

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -43,6 +43,20 @@
...
+    "*.js": [
+      "eslint --fix",
+      "git add"
+    ],
+    "*.ts": [
+      "prettier --write",
+      "git add"
+    ],
+    "package.json": [
+      "node cli.js",
```
Comment (keithamus) [inline package.json]: This is hilarious 😆

Comment (fisker) [reply package.json]: I tried to use stable `sort-package-json`, but seems npm can't install package with the same name? 

Should we swith to yarn?

Comment (keithamus) [reply package.json]: This is fine, it's like a little integration test for us 😄

Comment (fisker) [reply package.json]: Okay

**Pull request merged**